src/assets/svgs.tsx

```diff
@@ -51,3 +51,51 @@ export const ScaleBarIconTag = (
     </g>
   </svg>
 );
+
+export const FrameBackIconTag = (
+  <svg
+    id="e0a72a13-c21e-4c58-9a5e-a0ae842f6ad0"
```

Comment:
Does the SVG need the ID tag?

Reply:
Hm, I don't think so? I have typically just used the svgs exactly as provided in the website repo or UX drive folder, unless something needed to be changed. They seem to render fine with out the ids, do you think they should be removed?

Reply:
Might be better to, since if you have multiple copies of the same SVG on the website they'd all share the same ID. Probably doesn't matter too much